### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by wingnutter** [elementor/elementor#12998]
## Prerequisites

- [X ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
Make Elementor a truly accessible web design system.

**Describe the solution you'd like**
Add extra features to the One Click Accessibility plugin made by The Elementor / Pojo team.

a. The ability to change the space between letters.
b. The ability to use the Dyslexia font.
c. Play a small sound/beep when the accessibility menu is opened.

**Describe alternatives you've considered**
The UserWay accessibility plugin.

**Additional context**
We shouldn't be charging for this kind of thing, it should be freely available for all users of the web.

**Comment by Aviad-Herman** [comment]
Hi @wingnutter,
We really appreciate your feedback.

Accesability options are definitely something we are considering to implement.
This subject is actually really complex due to the different regulations in each country.

I'm marking this as a feature request to make sure we document the growing demand.

Thanks
